README.md

```diff
@@ -1,3 +1,5 @@
 # HYF Coach Bios
 
 A little hello and introduction from all of your coaches:
+
+Joel[JoeCamacho.md]
```

Comment:
That's not the right syntax for a link, it should be `[Joel](JoeChamacho.md)`.

Reply:
Sorry for the syntax error. It has been fixed